main.go

```diff
@@ -42,18 +46,24 @@ func main() {
 			log.Fatalf("expected line to have 2 elements, got %d for %v", len(line), line)
 		}
 
-
 		quiz = append(quiz, item{
 			Question: line[0],
 			Answer:   line[1],
 		})
 	}
 
-	resultsJSON, err := json.MarshalIndent(quiz, "", "    ")
+	possibleScore := len(quiz)
+	actualScore := 0
 
-	if err != nil {
-		log.Fatal(err)
+	for i, item := range quiz {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Printf("Problem #%v: %s = ", i, item.Question)
```

Comment:
%d instead of %v

Reply:
`%d` makes sense it is the base 10 representation of the value. What are the use cases for `%v`? I get it should be avoided for integers.

Reply:
I usually do `%+v` when printing maps or structs as you get the fields.  For primitive types I use the specific printing format.

Reply:
I actually noticed the difference in an example and a few lines down when I print the amount correct I use `%d`.